### ISSUE TITLE
Keine Anzeige im Oled Display 

### ISSUE THREAD

**Issue by u-meyer** [Bert-Tec/xbot_display#1]
Hallo,
erstmal finde ich es ein super Projekt .
Ich habe jetzt versucht ein ESP01 zu Flashen und ein Oled Display zu verbinden. Leider zeigt es nichts an. Bin wie in der Beschreibung vorgegangen. Auch den Widerstand habe ich eingelötet. Kann ich auch ohne Anschluss an den Scooter testen ob es funktioniert?  

Gruß 
Ulf

**Comment by Bert-Tec** [comment]
Ja, es sollte auch ohne Verbindung mit dem Scooter zumindest etwas anzeigen. Du kannst mal versuchen die zwei Kabel vom Display (SDA und SCL) zu tauschen. An sonsten läuft das Display mit einem Beispiel code?
